src/tasks/openCoverage.js

```diff
@@ -0,0 +1,27 @@
+const { platform } = require('os');
+const {
+  promises: { stat },
+} = require('fs');
+const { exec } = require('child_process');
+
+const file = 'coverage/lcov-report/index.html';
+
+(async () => {
```

Comment:
Can this be cleaned up in any way? Since we're doing the OS if statement two places, maybe we should use a function that checks OS and executes the correct command. I think that will make it cleaner and more readable.

What do you think?

Reply:
I would use a simple switch statement inside a function we can call.
```javascript
const open = () => {
  switch(platform()) {
    case 'win32':
      return exec(`start ${file}`);
    case 'darwin':
      return exec(`open ${file}`);
    default:
      return exec(`xdg-open ${file}`);
  }
}
```